test/src/test/java/org/corfudb/runtime/collections/CorfuQueueTest.java

```diff
@@ -67,6 +68,20 @@ public void basicQueueOrder() {
         assertThat(records.get(0).getRecordId().compareTo(records2.get(1).getRecordId())).isLessThan(0);
     }
 
+    @Test
+    public void queueWithSecondaryIndexCheck() {
+        CorfuQueue<String>
+                corfuQueue = new CorfuQueue<>(getDefaultRuntime(), "test", Serializers.JAVA,
+                CorfuTable.IndexRegistry.empty());
+
+        CorfuRecordId idC = corfuQueue.enqueue("c");
+        CorfuRecordId idB = corfuQueue.enqueue("b");
+        CorfuRecordId idA = corfuQueue.enqueue("a");
+
+        List<CorfuQueueRecord<String>> records = corfuQueue.entryList();
+        assertThat(records.size()).isEqualTo(3);
```

Comment:
I think this statement is giving the compilation issues. "3" a magic number

Reply:
thanks! fixed